src/components/events-table.js

```diff
@@ -59,25 +64,145 @@ function inputDataRow(processEvent) {
+                                </Button>
+                                <Menu
+                                    id="simple-menu"
+                                    anchorEl={anchorLevelEl}
+                                    keepMounted
+                                    open={Boolean(anchorLevelEl)}
+                                    onClose={handleCloseLevel}
+                                >
+                                    <MenuItem>
+                                        <TextField
+                                            id="levelTextField"
+                                            label={
+                                                <FormattedMessage id="filterOnLevel" />
+                                            }
+                                            type="text"
+                                            defaultValue={levelFilter}
+                                            InputLabelProps={{
+                                                shrink: true,
+                                            }}
+                                            onChange={handleLevelChange}
+                                        />
+                                    </MenuItem>
+                                </Menu>
+                            </span>
                         </TableCell>
                         <TableCell>
                             <FormattedMessage id="timestamp" />
                         </TableCell>
                         <TableCell>
                             <FormattedMessage id="eventDescription" />
+                            <span>
+                                <Button
+                                    aria-controls="simple-menu"
+                                    aria-haspopup="true"
+                                    onClick={handleLogMenuClick}
+                                    style={{
+                                        color:
+                                            logFilter === ''
+                                                ? 'inherit'
+                                                : '#3F51b5',
+                                    }}
+                                >
+                                    <FilterList />
+                                </Button>
+                                <Menu
+                                    id="simple-menu"
+                                    anchorEl={anchorLogEl}
+                                    keepMounted
+                                    open={Boolean(anchorLogEl)}
+                                    onClose={handleCloseLog}
+                                >
+                                    <MenuItem>
+                                        <TextField
+                                            id="levelTextField"
+                                            label={
+                                                <FormattedMessage id="filterOnLog" />
```

Comment:
Same, you have to define this label

Reply:
that's right. I did not split correctly my work in the multiple branch. The label is defined in the PR create a global view